examples/nlp/duplex_text_normalization/nn_wfst/en/electronic/normalize.py

```diff
@@ -12,16 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from nemo_text_processing.text_normalization.normalize import Normalizer
-from nemo_text_processing.text_normalization.token_parser import TokenParser
+try:
+    from nemo_text_processing.text_normalization.normalize import Normalizer
```

Comment:
For class imports that are used in-file like this, use the dummy class here - https://github.com/NVIDIA/NeMo/blob/2cc09425aba3e9b3cfdba43a3188eaef58227055/nemo/collections/nlp/models/language_modeling/megatron/gpt_model.py#L36

Class definition - https://github.com/NVIDIA/NeMo/blob/2cc09425aba3e9b3cfdba43a3188eaef58227055/nemo/collections/nlp/modules/common/megatron/utils.py#L36-L45